step-api/pom.xml

```diff
@@ -1,65 +1,38 @@
-<!--
-  Copyright (C) 2020, exense GmbH
-   
-  This file is part of STEP
-   
-  STEP is free software: you can redistribute it and/or modify
-  it under the terms of the GNU Affero General Public License as published by
-  the Free Software Foundation, either version 3 of the License, or
-  (at your option) any later version.
-   
-  STEP is distributed in the hope that it will be useful,
-  but WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  GNU Affero General Public License for more details.
-   
-  You should have received a copy of the GNU Affero General Public License
-  along with STEP.  If not, see <http://www.gnu.org/licenses/>.
--->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
 	<groupId>ch.exense.step</groupId>
 	<artifactId>step-api</artifactId>
-	<version>1.2.0-SNAPSHOT</version>
+	<version>0.0.0-SNAPSHOT</version>
 	<packaging>pom</packaging>
 
 	<name>Step API</name>
-	<description>
-		Step API
-	</description>
-	<url>http://exense.ch/</url>
+	<description>Step API</description>
 	<licenses>
 		<license>
 			<name>The GNU Affero General Public License 3.0</name>
 			<url>http://www.gnu.org/licenses/</url>
 		</license>
 	</licenses>
-	<developers>
```

Comment:
Same, I have the feeling that meta data were removed a bit everywhere (i.e. other refactored repo too).